components/style/values.rs

```diff
@@ -639,6 +639,7 @@ pub mod specified {
     }
 
     impl ToCss for Calc {
+        #[allow(unused_assignments)]
```

Comment:
The `serialize!` macro below triggers ``warning: value assigned to `first_value` is never read`` in the last iteration of of the `$( … )*` macro loop, be refactoring the macro to not do that is annoying.